addons/gdUnit4/src/update/GdUnitUpdateNotify.gd

```diff
@@ -5,7 +5,7 @@ signal request_completed(response)
 
 const GdMarkDownReader = preload("res://addons/gdUnit4/src/update/GdMarkDownReader.gd")
 const GdUnitUpdateClient = preload("res://addons/gdUnit4/src/update/GdUnitUpdateClient.gd")
-const spinner_icon := "res://addons/gdUnit4/src/ui/assets/spinner.tres"
+const spinner_icon := preload("res://addons/gdUnit4/src/ui/assets/spinner.tres")
```

Comment:
Do not replace the string representation with the resource. 
This const variable needs to be a String, it is used as message to build markdown text